antenna/throttler.py

```diff
@@ -277,86 +277,79 @@ def match_unsupported_product(throttler, data):
         percentage=None
     ),
 
-    # Reject all unsupported products; this does nothing if the list of
-    # supported products is empty
+    # Reject crash reports for unsupported products; this does nothing if the
+    # list of supported products is empty
     Rule(
         rule_name='unsupported_product',
         key='*',
         condition=match_unsupported_product,
         percentage=None
     ),
 
-    # 100% of crashes that have a comment
+    # Accept 100% of crash reports submitted through about:crashes
+    Rule(
+        rule_name='throttleable_0',
+        key='Throttleable',
+        condition=lambda throttler, x: x == '0',
+        percentage=100
+    ),
+
+    # Accept 100% of crash reports that have a comment
     Rule(
         rule_name='has_comments',
         key='Comments',
         condition=always_match,
         percentage=100
     ),
 
-    # 100% of crashes that have an email address with at least an @
+    # Accept 100% of crash reports that have an email address with at least an @
     Rule(
         rule_name='has_email',
         key='Email',
         condition=lambda throttler, x: x and '@' in x,
         percentage=100
     ),
 
-    # 100% of all ReleaseChannel=aurora, beta, esr channels
+    # Accept 100% of crash reports in ReleaseChannel=aurora, beta, esr channels
     Rule(
         rule_name='is_alpha_beta_esr',
         key='ReleaseChannel',
         condition=lambda throttler, x: x in ('aurora', 'beta', 'esr'),
         percentage=100
     ),
 
-    # 100% of all ReleaseChannel=nightly
+    # Accept 100% of crash reports in ReleaseChannel=nightly
     Rule(
         rule_name='is_nightly',
         key='ReleaseChannel',
         condition=lambda throttler, x: x.startswith('nightly'),
         percentage=100
     ),
```

Comment:
I kept the above two rules because I (ab)use the dashboard showing throttle rules that got triggered to figure out what happened when we get spikes of incoming crashes. I shouldn't do that. I should use different data. Can't really use super search because I want to view a timeseries--not counts. I'll think about this more at some point. For now, I'm going to leave these two rules in.